src/script/_lib/mod/vco-commons.lua

```diff
@@ -95,23 +95,23 @@ end
 -- ITEMS --
 
 function vlc.items:add_item_to_faction_inventory(item_key, faction_key)
-    local faction = cm:get_faction(faction_key);
-    cm:add_ancillary_to_faction(faction, item_key, false);
+	local faction = cm:get_faction(faction_key);
+	cm:add_ancillary_to_faction(faction, item_key, false);
 end
 
 -- NAGASH BOOKS --
 
-function vlc.nagash_books:add(faction_id, book_number)
```

Comment:
Using **faction_id** instead of **faction_key** was intentional. It was a way to state that there is no need to use a real faction key, and that something shorter can be used. For example, using "karl" for the Empire, or something similar.

Reply:
One example was using **tmb_ark** instead of **wh2_dlc09_tmb_followers_of_nagash**. Not a big deal, but there was no need to replace it all imo.